<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: rgba(0, 0, 0, 0.5);
border-radius: 3px;}
#change{
border:0px;
border-radius:5px;
background-color:rgb(94, 193, 130);
font-family:Roboto;
color:white;
font-size:16px;
	
color: #333;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 80px;
background-color: rgba(255,255,255, .5);
background-color:rgb(58, 120, 170);
}

#change:hover {
color:black;
background: rgb(58, 86, 132)


}

#change:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;
color:white</string>
    </property>
    <property name="text">
     <string>Current Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>New Password:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #494949, stop: 1 #ffffff);
background-color: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 25 20px  &quot;Ubuntu Light&quot;;
text-decoration: bold;
background-color: transparent;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Jaedong</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="LIBPRO.qrc">:/barIcon/Resources/icon/user (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="LIBPRO.qrc">
      <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;

color:rgb(55,55,55);
color:white;</string>
    </property>
    <property name="text">
     <string>Confirm new password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;
border-radius:0px;

color:white;
font-family:Roboto</string>
    </property>
    <property name="text">
     <string>CHANGE PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldPassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="LIBPRO.qrc">
      <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="LIBPRO.qrc">
      <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-family:Roboto;
background-color:transparent;
color:white;
font-weight:500;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>511</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color: rgba(0, 0, 0, 0.5);
border-radius: 3px;}
#change{
border:0px;
border-radius:5px;
background-color:rgb(94, 193, 130);
font-family:Roboto;
color:white;
font-size:16px;
	
color: #333;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 80px;
background-color: rgba(255,255,255, .5);
background-color:rgb(58, 120, 170);
}

#change:hover {
color:black;
background: rgb(58, 86, 132)


}

#change:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;
color:white</string>
     </property>
     <property name="text">
      <string>Current Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>New Password:</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #494949, stop: 1 #ffffff);
background-color: transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="username_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font: 25 20px  &quot;Ubuntu Light&quot;;
text-decoration: bold;
background-color: transparent;
text-align:center;</string>
      </property>
      <property name="text">
       <string>Jaedong</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LIBPRO.qrc">:/barIcon/Resources/icon/user (1).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn1_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="LIBPRO.qrc">
       <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-family:Roboto;
background-color: transparent;

color:rgb(55,55,55);
color:white;</string>
     </property>
     <property name="text">
      <string>Confirm new password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
border-radius:0px;

color:white;
font-family:Roboto</string>
     </property>
     <property name="text">
      <string>CHANGE PASSWORD</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldPassword_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="LIBPRO.qrc">
       <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPassword_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(219, 219, 219);
background-color:transparent;
border-radius: 2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="LIBPRO.qrc">
       <normaloff>:/icon/Resources/icon/key (1).png</normaloff>:/icon/Resources/icon/key (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirm_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid rgb(250, 250, 250);
border-left: 0px solid rgb(250, 250, 250);
border-right: 0px solid rgb(250, 250, 250);
border-bottom: 1px solid white;
border-radius:0px;
margin:5px;
background-color: transparent;
font-family:Roboto;
font-size:16px;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="error_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>441</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-family:Roboto;
background-color:transparent;
color:white;
font-weight:500;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="LIBPRO.qrc"/>
 </resources>
 <connections/>
</ui>
